docs/API/panel/heatmap/index.md

```diff
@@ -515,9 +515,10 @@ Controls frame rows options
 withShowValue(value)
 ```
 
-TODO docs
-
-Accepted values for `value` are "auto", "never", "always"
+| *{
+	layout: ui.HeatmapCellLayout & "auto" // TODO: fix after remove when https://github.com/grafana/cuetsy/issues/74 is fixed
+}
```

Comment:
Interesting artifact propagating through the docstring, prolly need to prevent that somehow.